crates/matrix-sdk/src/widget/machine/incoming.rs

```diff
@@ -47,8 +47,12 @@ pub(crate) enum IncomingMessage {
     /// The `MatrixDriver` notified the `WidgetMachine` of a new Matrix event.
     ///
     /// This means that the machine previously subscribed to some events
-    /// ([`crate::widget::Action::Subscribe`] request).
+    /// ([`crate::widget::Action::SubscribeTimeline`] request).
     MatrixEventReceived(Raw<AnyTimelineEvent>),
+
+    /// The `MatrixDriver` notified the `WidgetMachine` of a new to_device
```

Comment:
```suggestion
    /// The `MatrixDriver` notified the `WidgetMachine` of a new to-device
```

Reply:
fixed, found a couple more in the widget mod https://github.com/matrix-org/matrix-rust-sdk/pull/4963/commits/f248d2792104e4d1385e49e22b6d56ae98cd52a5